added google services connection

itaycsguy opened the pull request:
verify you are not connected to some google account or to firebase at first

itaycsguy committed:
added google services connection

Morag140 commented on app/src/main/java/app/itaycsguy/musiciansaidb/FirebaseAuth.kt:
```diff
@@ -5,16 +5,32 @@ import com.google.android.gms.auth.api.signin.GoogleSignInAccount
+                return true
+            }
+            _gso = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN).requestIdToken(R.string.default_web_client_id.toString()).requestEmail().build()
+            CustomSnackBar.make(_act,"Connected to firebase!")
```
I don't think the users should "know".

Morag140 commented on app/src/main/java/app/itaycsguy/musiciansaidb/FirebaseAuth.kt:
```diff
@@ -5,16 +5,32 @@ import com.google.android.gms.auth.api.signin.GoogleSignInAccount
+            CustomSnackBar.make(_act,"Connected to firebase!")
+            true
+        } catch(e : java.lang.Exception){
+            CustomSnackBar.make(_act,"Could not connect to firebase.")
```
Enough to say : could not connect to online database.

Morag140 reviewed (approved):
I didnt check it properly...because i dont know how to remove firebase from my studio

itaycsguy committed:
1. added firebase manual connection + made permission checking for signApp class
2. check the application for all its' sections and it is running well and ready to production from my simulation
